download.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Downloads
+subtitle: Select the SD card image file for your hardware below
```

Comment:
We generally haven't referred to the page itself in the subtitle, so I think this would be better without the "below"